packages/text-area/src/TextArea.tsx

```diff
@@ -214,14 +214,23 @@ const TextArea: React.ComponentType<
         </Label>
       )}
       {description && (
-        <Description darkMode={darkMode} disabled={disabled}>
+        <Description
+          darkMode={darkMode}
+          disabled={disabled}
+          className={cx({
+            [css`
+              padding-bottom: 4px;
```

Comment:
It does look like this component already has `padding-bottom: 4px` for both modes
<img width="166" alt="Screen Shot 2022-01-24 at 5 03 22 PM" src="https://user-images.githubusercontent.com/5740159/150872538-3c84971c-f0cc-4c03-9007-7bca3c7d2430.png">